packages/opentelemetry-sdk-trace-base/test/common/export/InMemorySpanExporter.test.ts

```diff
@@ -93,6 +93,21 @@ describe('InMemorySpanExporter', () => {
     });
   });
 
+  it('should reset spans when reset is called', async () => {
```

Comment:
we're not `await`ing here so it's fine for this to not be `async` :slightly_smiling_face: 
```suggestion
  it('should reset spans when reset is called', () => {
```